docs/reference/monitoring/troubleshooting.asciidoc

```diff
@@ -27,3 +30,14 @@ monitoring data by using {metricbeat} the indices have `-mb` in their names. If
 the indices do not exist, review your configuration. For example, see
 <<monitoring-production>>.
 
+[[monitoring-troubleshooting-uuid]]
+=== Some monitoring data is missing from {kib}
+
+*Symptoms*:
+The information about some nodes, instances, or Beats in your cluster are
+missing from the *Stack Monitoring* page in {kib}.
+
+*Resolution*:
+Verify that the missing items have unique UUIDs. Each {es} node, {ls} node,
+{kib} instance, and Beat is considered unique based on its persistent UUID,
```

Comment:
Perhaps @dedemorton can weigh in here: what term do we like to use for an individual running Beat process? An instance? A node? Something else?

Reply:
I prefer instance.